Private/New-Dictionary.ps1

```diff
@@ -125,15 +125,15 @@ function New-Dictionary {
         #     FixIt = {Write-Output 'Add code to fix the vulnerable configuration.'}
         #     ReferenceUrls = ''
         # },
-        # [VulnerableConfigurationItem]@{
-        #     Name = 'ESC11'
-        #     Category = 'Escalation Path'
-        #     Subcategory = ''
-        #     Summary = ''
-        #     FindIt =  {Find-ESC11}
-        #     FixIt = {Write-Output 'Add code to fix the vulnerable configuration.'}
-        #     ReferenceUrls = ''
-        # },
+        [VulnerableConfigurationItem]@{
```

Comment:
Changes to the dictionary should include a bump in the dictionary version attribute on line 24, which is currently: `static [string] $Version = '2023.10.01.000'`.

Reply:
Fixed!